### ISSUE TITLE
[ICC][Lich King] Val'Kyr Shadowguard Life Siphon

### ISSUE THREAD

**Issue by Dibbarra** [dalaranwow/dalaran-wow#5413]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Valkyr casting Life Siphon target the person who has the highest threat and casts life siphon only on that target.
**Expected behaviour**: Tell us what should happen instead
Life Siphon should be cast on random raid members so the UH DK (on 25man) or healer (on 10man) are not shredded by valks during phase 2.
**Steps to reproduce the problem**:

1. Do LoD attempts
2. Survive 2 waves of Valkyrs
3. Watch sevendk be #1 on damage taken charts.

**Include proofs for this behaviour**
http://www.wowhead.com/npc=36609/valkyr-shadowguard
"On heroic mode they will drop their target when they reach 50% health then fly back up above the platform and cast leeches on random members of the raid until the phase 3 transition. While they're leeching, they are tauntable similarly to the Faction Champs fight, so the tanks can keep taunting to keep the leeches on themselves rather than on random raid members."
--

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
73488 - Spell ID
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
![image](https://user-images.githubusercontent.com/8012118/39025385-9be25a30-4414-11e8-9d87-e52e6aa391d3.png)

**Comment by Rushor** [comment]
In fact i'm not sure if the creature should cast the spell on randoms or only on the top aggro victim. Any proofs for this? 

I will add fixes for both cases.

**Comment by Dibbarra** [comment]
Guide videos like https://www.youtube.com/watch?v=h1x9MErf5NA & https://www.youtube.com/watch?v=qog5PgpBUQY & forum posts like https://www.mmo-champion.com/threads/694764-Guide-to-Heroic-ICC
seem to  imply that targeting is random, but that you can taunt to make them focus the tank (or a specific person).
